selfdrive/radar/nidec/interface.py

```diff
@@ -1,14 +1,21 @@
 #!/usr/bin/env python
+import os
 import numpy as np
+
 from selfdrive.car.honda.can_parser import CANParser
+from selfdrive.can.parser import CANParser as CANParserC
```

Comment:
The openpilot tests still relies on it. Will be removed when the openpilot tests are replaced.